numpy/lib/utils.py

```diff
@@ -19,21 +19,17 @@
 def get_include():
     """
     Return the directory that contains the NumPy \\*.h header files.
-
```

Comment:
Please undo all the deletion of blank lines in this PR. These blank lines are correct and necessary for rendering with Sphinx.

Reply:
Sorry this was unintentional. Its been fixed now.